www/docs/regmem/index.php

```diff
@@ -58,8 +58,9 @@ function person_history($p) {
     $earliest = $files[0];
     foreach ($files as $_) {
         $file = file_get_contents($_);
+        $iso = preg_replace("#$dir/regmem(.*?)\.xml#", '$1', $_);
```

Comment:
I think I'd prefer $date which makes it a bit more obvious what's being captured.